spec.html

```diff
@@ -38846,7 +38846,10 @@ <h1>%AsyncFromSyncIteratorPrototype%.next ( _value_ )</h1>
             1. Assert: Type(_O_) is Object and _O_ has a [[SyncIteratorRecord]] internal slot.
```

Comment:
Should we mark `value` parameter as optional, especially since it is not observable via `length`?

Reply:
Yes, that seems reasonable.

Reply:
Seems like the recent change brought back empty lists &laquo; &raquo;.